src/olympia/amo/admin.py

```diff
@@ -180,7 +182,7 @@ def get_query_string(self, new_params=None, remove=None):
 
 class AMOModelAdmin(admin.ModelAdmin):
     class Media:
-        js = (
+        js = ((vite_hmr_client(),) if settings.DEV_MODE else ()) + (
```

Comment:
mmm, doesn't this return a full script tag (like `"<script ... >") rather than path?

Reply:
You're right, but it works 🤷 

<img width="443" alt="image" src="https://github.com/user-attachments/assets/4f94b911-3b51-426b-ba0d-65c2d0da188f" />

Reply:
🙈

Reply:
Confirmed, removing the function removes the client, and adding it back adds it working as expected.